install.sh

```diff
@@ -521,6 +536,90 @@ do_install() {
 			echo_docker_as_nonroot
 			exit 0
 			;;
+			
+		sles)
+			arch=$(uname -m)
+			if [[ $arch != "s390x" ]]; then
+				echo "Docker-CE is not supported on SLES for $arch!!!"
+				exit 1
+			else
+				echo "Installing Docker-CE on $arch!"
+			fi
+
+			sles_repo="$DOWNLOAD_URL/linux/$lsb_dist/$REPO_FILE"
+			opensuse_repo="https://download.opensuse.org/repositories/security:SELinux/SLE_15_SP2/security:SELinux.repo"
+			if ! curl -Ifs "$sles_repo" > /dev/null; then
+				echo "Error: Unable to curl repository file $sles_repo, is it valid?"
+				exit 1
+			fi
+			pre_reqs="ca-certificates curl libseccomp2 iptables awk"
```

Comment:
(not for this PR); looks like we need to add `awk` to some of the other `pre_reqs` as well (as it's used in this script for other distress as well).

- the `libseccomp2` here looks to be "temporarily", until the package dependencies for `docker-ce` and `containerd.io` are updated, correct?
- is the `iptables` needed here? I see it's already a dependency in the RPM; https://github.com/docker/docker-ce-packaging/blob/20.10/rpm/SPECS/docker-ce.spec#L26

Reply:
Yes, once `docker-ce` and `containerd.io` are updated we can remove libseccomp2 from here.
Removed the `iptables` dependency.